Assets/UPM/Console/CommandDiscovery.cs

```diff
@@ -23,7 +27,7 @@ public static CommandDiscovery Build(params Assembly[] assemblies)
                 {
                     foreach (var attribute in method.GetCustomAttributes(false).OfType<CommandAttribute>())
                     {
-                        var command = new Command(method, method.Name);
+                        var command = new Command(method, attribute.CustomName == "" ? method.Name : attribute.CustomName); // added so we can have a custom Name :)
```

Comment:
`String.IsNullOrEmpty(attribute.CustomName)` might be a bit more clear and cover the null case.